doc/source/getting_started/running_mapdl.rst

```diff
@@ -21,6 +24,20 @@ installer options can change, see the following figure for reference.
     :width: 400pt
 
 
+If you prefer, you can use Docker to avoid having to install MAPDL
+locally.
+This is specially convenient if you are using a non-supported platform such
+as MacOS.
+Visit :ref:`pymapdl_docker` for more information.
```

Comment:
```suggestion
as MacOS. For more information, see :ref:`pymapdl_docker` .
```

Reply:
I don't think it makes a difference when we render.